n_cjson_helpers.c

```diff
@@ -355,19 +383,23 @@ bool JAddBinaryToObject(J *req, const char *fieldName, const void *binaryData, u
     return true;
 }
 
-//**************************************************************************/
+
 /*!
-    @brief  Get binary from an object that is expected to be a Base64-encoded string.
-    @param   rsp The JSON object containing the  field.
-    @param   fieldName The field to get data from.
-    @param   retBinaryData The binary data object allocated.  (Use standard "free" method to free it.)
-			 Note that, as a convenience to the caller in case the "binary data" is actually a string,
-			 one byte extra is allocated in the return buffer which is filled with '\0'.  This byte
-			 is not included in the retBinaryDataLen length.
-    @param   retBinaryDataLen The length of the binary data.
-    @returns bool. Whether the binary data was allocated and returned.
-*/
-/**************************************************************************/
+ @brief Decode a Base64-encoded string field in a JSON object and return the
+        decoded bytes.
+
+ @param rsp The JSON object.
+ @param fieldName The name of the field.
+ @param retBinaryData A pointer to a buffer to hold the decoded bytes.
+ @param retBinaryDataLen A pointer to a number to hold the length of the decoded
+        bytes.
```

Comment:
```suggestion
```suggestion
 @param retBinaryDataLen A pointer to an unsigned integer, used to hold the
        length of the decoded bytes.
```